docs/whats-new/dotnet-7-docs.md

```diff
@@ -0,0 +1,295 @@
+- [Fields (C# Programming Guide)](../csharp/programming-guide/classes-and-structs/fields.md) - C# 11: required members
+- [How to declare and use read write properties (C# Programming Guide)](../csharp/programming-guide/classes-and-structs/how-to-declare-and-use-read-write-properties.md) - C# 11: required members
+- [How to implement a lightweight class with auto-implemented properties (C# Programming Guide)](../csharp/programming-guide/classes-and-structs/how-to-implement-a-lightweight-class-with-auto-implemented-properties.md) - C# 11: required members
+- [Properties (C# Programming Guide)](../csharp/programming-guide/classes-and-structs/properties.md) - C# 11: required members
+- [Using Properties (C# Programming Guide)](../csharp/programming-guide/classes-and-structs/using-properties.md) - C# 11: required members
+- [Properties](../csharp/properties.md) - C# 11: required members
+- [Constraints on type parameters (C# Programming Guide)](../csharp/programming-guide/generics/constraints-on-type-parameters.md) - Add information on static virtual interface members
+- [Generic Interfaces (C# Programming Guide)](../csharp/programming-guide/generics/generic-interfaces.md) - Add information on static virtual interface members
+- [Built-in types (C# reference)](../csharp/language-reference/builtin-types/built-in-types.md) - Add updates for numeric IntPtr
+- [Built-in numeric conversions (C# reference)](../csharp/language-reference/builtin-types/numeric-conversions.md) - Add updates for numeric IntPtr
+- [Arithmetic operators (C# reference)](../csharp/language-reference/operators/arithmetic-operators.md) - Operator updates to support generic math
+- [Determine caller information using attributes interpreted by the C# compiler](../csharp/language-reference/attributes/caller-information.md) - Extended nameof parameter scope
+- [Attributes for null-state static analysis interpreted by the C# compiler](../csharp/language-reference/attributes/nullable-analysis.md) - Extended nameof parameter scope
+- [delegate operator (C# reference)](../csharp/language-reference/operators/delegate-operator.md) - Document new method group conversion
+- [Generics and Attributes (C# Programming Guide)](../csharp/programming-guide/generics/generics-and-attributes.md) - generic attributes are allowed in C# 11
+
+## F# language
+
+### Updated articles
+
+- [F# docs - get started, tutorials, reference.](../fsharp/index.yml) - Merge .NET 7 release branch into main
+- [What's new in F# 6](../fsharp/whats-new/fsharp-6.md)
+  - Merge .NET 7 release branch into main
+  - Retire the "What's new in C# 8" article
+- [What's new in F# 5](../fsharp/whats-new/fsharp-50.md) - Retire the "What's new in C# 8" article
+
+## Microsoft Orleans
+
+### New articles
+
+- [What's new in Orleans 7.0](../orleans/whats-new-in-orleans.md)
+
+### Updated articles
+
+- [Orleans transactions](../orleans/grains/transactions.md) - Merge .NET 7 release branch into main
```

Comment:
Should this be included?  And should it have a different note about the change?

Reply:
```suggestion
- [Orleans transactions](../orleans/grains/transactions.md) - Describe new APIs for .NET 7
```